skimage/morphology/_flood_fill.py

```diff
@@ -236,13 +234,16 @@ def flood(image, seed_point, *, selem=None, connectivity=None, tolerance=None):
     except TypeError:
         seed_point = (seed_point,)
 
+    seed_value = image[seed_point]
```

Comment:
@FirefoxMetzger actually, this will fail with out-of-range seed points, so I think we're covered. =)